sdk/include/stdlib.h

```diff
@@ -52,15 +85,15 @@ void *__cheri_compartment("alloc") heap_allocate(size_t size, Timeout *timeout);
  * Memory returned from this interface is guaranteed to be zeroed.
  */
 void *__cheri_compartment("alloc")
-  heap_allocate_array(size_t nmemb, size_t size, Timeout *timeout);
+  heap_allocate_array(struct SObjStruct *heapCapability, size_t nmemb, size_t size, Timeout *timeout);
```

Comment:
Do you want to have an `ifdef __cplusplus` overload for this and for `heap_free`, too?

Reply:
This is going away, it was just there so I could start to get things working without updating all of the callers.